src/test/java/jalau/at18/Azul/BagTest.java

```diff
@@ -0,0 +1,39 @@
+package jalau.at18.azul;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotEquals;
+
+import java.util.ArrayList;
+
+import org.junit.Test;
+
+
+public class BagTest {
+
+    @Test
+    public void shouldVerifyThereIsBag() {
+        ArrayList<Tile> aBag = new ArrayList<>();
+        Bag myBag = new Bag();
+        Tile[] color = {Tile.RED, Tile.YELLOW, Tile.DARK, Tile.BLUE, Tile.WHITE};
```

Comment:
This is technically replicating the behavior of the `fillBagWithColors` method. The idea of the test is not to replicate logic of the function being tested, but setting the input and output expected.

In this case, based on the name of the method, you need to define what is the expected status after "bag is filled with colors". Say, you can assert that `myBag` has `Tile.RED` color, `Tile.YELLOW` color an so on.

Reply:
Changing the test name and structure, now it will fill a list that match the list in the Bag Class.